simplesat/tests/test_compute_dependencies.py

```diff
@@ -109,3 +110,21 @@ def test_chained_dependencies_transitive(self):
         deps = compute_reverse_dependencies(self.repos, requirement,
                                             transitive=True)
         self.assertEqual(deps, set(expected_deps))
+
+
+class TestComputeLeafPackages(unittest.TestCase):
+
+    def setUp(self):
+        repo_0 = Repository(packages_from_definition(PACKAGE_DEF_0))
+        repo_1 = Repository(packages_from_definition(PACKAGE_DEF_1))
+        self.repos = [repo_0, repo_1]
+
+    def test_simple(self):
```

Comment:
Can you make sure to cover the case where the same package is available in multiple repositories?